webhook_to_fedora_messaging/models/message.py

```diff
@@ -0,0 +1,16 @@
+# SPDX-FileCopyrightText: Contributors to the Fedora Project
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from sqlalchemy import Column, Integer, UnicodeText, ForeignKey
+from webhook_to_fedora_messaging.models.util import UUIDCreatableMixin, CreatableMixin
+
+from webhook_to_fedora_messaging.database import Base
+
+
+class Message(Base, UUIDCreatableMixin, CreatableMixin):
+    __tablename__ = "messages"
+
+    id = Column(Integer, primary_key=True, nullable=False)
+    service_id = Column(Integer, ForeignKey("service.id", ondelete="CASCADE"), unique=False, nullable=False)
+    digest = Column(UnicodeText, nullable=False)
```

Comment:
I'm not sure it's useful to store emitted message, this looks like some sort of log.

Reply:
I cannot think of a strong reason why we would want to store the digest (a diminutive hash that does not burden the database with the full load of the message and/or compromise the message privacy by revealing its contents) but FWIW, it helps us distinguish between the messages that came from W2FM and the ones that did not.

As to why we would need that, I would need to scratch my head a bit more for that I suppose.

Reply:
Yeah let's just drop the `messages` table, I don't think we'll need it.